client/src/Components/ReviewsList.jsx

```diff
@@ -32,20 +51,30 @@ class ReviewsList extends React.Component {
         }
         return res.json();
       })
-      .then((reviews) => {
+      .then((allReviews) => {
         // Ensure our Reviews are sorted chronologically
-        reviews.sort((reviewA, reviewB) => (
+        allReviews.sort((reviewA, reviewB) => (
           new Date(reviewB.createdAt) - new Date(reviewA.createdAt)
         ));
-        this.setState({ reviews });
+
+        const reviewLimit = allReviews.length < 10 ? allReviews.length : 10;
+        this.setState({ 
+          allReviews,
+          displayedReviews: allReviews.slice(0, reviewLimit)
+        });
       })
       .catch((err) => {
         console.error(err);
       });
   }
 
   render() {
-    const Reviews = this.state.reviews.map(reviewRecord => (
+    const loadMoreReviewsButton = this.state.displayedReviews.length < this.state.allReviews.length ? (
+        <button className="load-button" onClick={ this.displayMoreReviews } >Load More Reviews</button>
+      ) : (
```

Comment:
could line 74 - 76 be 1 line?

Reply:
It could but I prefer this style for when dealing with multi-line ternary operations so you can see the two results at a quick glance and they're both consistent with each other.